lib/config/default.config.js

```diff
@@ -105,6 +105,7 @@ module.exports = {
 
   security: {
     restrictedProfileIds: ['default'],
+    supportCookieAuthentication: false,
```

Comment:
This should be true by default IMHO
```suggestion
    supportCookieAuthentication: true,
```

Reply:
IDK the problem with this option is that the CORS Header cannot be defined as a wildcard, so if we defined it to true by default, we can't set the `http.accessControlAllowOrigin` to `*`